src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage.js

```diff
@@ -1,78 +1,127 @@
-import React from 'react';
+import React, {useEffect, useRef} from 'react';
+import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
-import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import CONST from '../../../../CONST';
 import ONYXKEYS from '../../../../ONYXKEYS';
+import ROUTES from '../../../../ROUTES';
 import MoneyRequestParticipantsSplitSelector from './MoneyRequestParticipantsSplitSelector';
 import MoneyRequestParticipantsSelector from './MoneyRequestParticipantsSelector';
 import styles from '../../../../styles/styles';
-import FullScreenLoadingIndicator from '../../../../components/FullscreenLoadingIndicator';
-import optionPropTypes from '../../../../components/optionPropTypes';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import compose from '../../../../libs/compose';
+import * as DeviceCapabilities from '../../../../libs/DeviceCapabilities';
+import HeaderWithBackButton from '../../../../components/HeaderWithBackButton';
+import * as IOU from '../../../../libs/actions/IOU';
 
 const propTypes = {
-    /** Callback to inform parent modal of success */
-    onStepComplete: PropTypes.func.isRequired,
-
-    /** Should we request a single or multiple participant selection from user */
-    hasMultipleParticipants: PropTypes.bool.isRequired,
-
-    /** Callback to add participants in MoneyRequestModal */
-    onAddParticipants: PropTypes.func.isRequired,
-
-    /** Selected participants from MoneyRequestModal with login */
-    participants: PropTypes.arrayOf(optionPropTypes),
-
-    /* Onyx Props */
-
-    /** Holds data related to IOU view state, rather than the underlying IOU data. */
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
     iou: PropTypes.shape({
-        /** Whether or not the IOU step is loading (retrieving participants) */
-        loading: PropTypes.bool,
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        participants: PropTypes.arrayOf(
+            PropTypes.shape({
+                accountID: PropTypes.number,
```

Comment:
Could you please come back and update this code to include comments on all these properties?

> Add descriptions to all propTypes using a single line comment above the definition. No need to document the types, but add some context for each property so that other developers understand the intended use.

https://github.com/Expensify/Style-Guides/blob/main/react.md#things-to-do